common/src/View/MapViewBase.cpp

```diff
@@ -244,7 +245,8 @@ namespace TrenchBroom {
         }
 
         void MapViewBase::preferenceDidChange(const IO::Path& path) {
-			updateAcceleratorTable();
+            updateAcceleratorTable();
+            fontManager().clearCache();
```

Comment:
Now the font cache is cleared every time a preference changes. You can limit this to the font size preference by inspecting the path:

```
if (path == Preference::RendererFontSize) {
    fontManager().clearCache();
}
```

Reply:
Note that the path is the path of the preference itself within the preference store, e.g. `Renderer/Font size` in this case. It is used to find out which preference has changed when this notification is received.

Reply:
Note that the path is the path of the preference itself within the preference store, e.g. `Renderer/Font size` in this case. It is used to find out which preference has changed when this notification is received.